airflow/kubernetes/kube_client.py

```diff
@@ -19,25 +19,18 @@
 
 import logging
 
+import urllib3
+
 from airflow.configuration import conf
 
 log = logging.getLogger(__name__)
 
 try:
     from kubernetes import client, config
-    from kubernetes.client import Configuration
+    from kubernetes.client import ApiClient, Configuration
     from kubernetes.client.rest import ApiException
 
     has_kubernetes = True
-
-    def _disable_verify_ssl() -> None:
-        if hasattr(Configuration, 'get_default_copy'):
-            configuration = Configuration.get_default_copy()
-        else:
-            configuration = Configuration()
-        configuration.verify_ssl = False
-        Configuration.set_default(configuration)
```

Comment:
Is it ok to remove this `set_default` and only rely on every other code path going through `get_kube_client`?

I see in `pod_generator` and `TaskInstance` creates `ApiClient` in many places, but only uses it for offline operations.

It's also created by `hooks.kubernetes`, haven't looked into what that does yet.

-----

Maybe it's safer to keep it this way and incorporate the new config using this same method?